ynr/apps/people/helpers.py

```diff
@@ -120,6 +121,12 @@ def clean_twitter_username(username):
     return username
 
 
+def clean_linkedin_url(url):
+    if not re.match(r"^https?://(www\.)?linkedin\.com/in/[\w-]+/?$", url):
```

Comment:
Can you update this as per the insta PR to use `urlparse` to match the domain(s) and path?

Reply:
Sorry, I thought I had! Will do.